designs/administrator/future-openshift/resource-related-quick-starts/index.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Surfacing resource-related quick starts
+
+## Introduction
+Sometimes quick starts can be closely associated with a resource type, for example, a virtual machine or a pipeline. With that in mind, it became important to surface quick starts on that resource's page to improve the flow of both creating a resource and using a resource. This design demonstrates how to do so.
+
+## Design summary
+The design is broken up into scenarios:
+- [Populated resource list views](#populated-resource-list-views)
+- [Empty resource list views](#empty-resource-list-views)
+
+Whether the list view is populated or not, the general pattern of this design includes surfacing a dismissible label to make it clear that a quick start is available within the page heading.
+![general-design](img/general-design.png)
+
+The label is more like a hint and can be hidden, which would then hide **all** resource-related quick start labels for that user.
+![general-design-hide](img/general-design-hide.png)
+
+If there are multiple quick starts associated with a resource, they should be shown in a side panel, where the order logic should be as follows:
+- In progress quick starts
+- Not started quick starts
+- Do not show completed quick starts
+![side-panel](img/side-panel.png)
+
+At the end of the list, there should be a link to view all quick starts associated with that resource. The button should take the user to the quick starts catalog page that is filtered by the resource type.
+![side-panel-cta](img/side-panel-cta.png)
+
+## Empty resource list views
+If a quick start exists that will help the user create a resource type, it should be made available within the empty state of the list view.
+![empty-list](img/empty-list.png)
+
+Selecting the button will immediately start the quick start on the page.
```

Comment:
Thanks for drawing out the sidebar interactions. For the operator-equivilent design, I will likely just reference this design for the sidebar behaviors, since I think they should be consistent.

Reply:
Will the quick start immediately start, or will the panel open and then the user has to click "Start"? If the latter, I recommend that we say: 

"Clicking the button opens the quick start in a side panel."

Reply:
@abigaeljamie They'll have to click "Start" (or whatever the CTA is at that point) in order to use it!

Reply:
Thanks, @gdoyle1 ! Then in that case, I'd say: 

"Clicking the button opens the quick start in a side panel. The user can then click the side panel button to begin the quick start."